hack/.golint_failures

```diff
@@ -449,12 +449,6 @@ staging/src/k8s.io/api/storage/v1alpha1
 staging/src/k8s.io/api/storage/v1beta1
 staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/apis/cr
 staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/apis/cr/v1
-staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/clientset/versioned
```

Comment:
This seems unrelated to the rest of the PR?

Reply:
Run hack/update-all.sh don't contain file in the apiextensions-apiserver/examples, so I run the script in apiextensions-apiserver/examples manually.